README.md

```diff
@@ -173,6 +173,7 @@ Supported settings in EVCC configuration are given below:
 | sdpRetryCycles         | `1`                                                          | Indicates how often shall SDP (SECC Discovery Protocol) retries happen before reverting to using nominal duty cycle PWM-based charging |
 | maxContractCerts       | `3`                                                          | Maximum amount of contract certificates the EV stores.                                                                                 |
 | maxSupportingPoints    | `1024`                                                       | Indicates the maximum number of entries the EVCC supports within the sub-elements of a ScheduleTuple                                   |
+| isAliveAfterSession    | `True`                                                       | Whether or not the EVCC keeps running after a charging session is finished                                                             |
```

Comment:
Apologies I didn't clarify this under issues.  As the user can't proceed until the session is killed anyway, I don't see much value in making this a configurable field. The behaviour you have added to gracefully exit is the only change required. Do you have a use case in mind where keeping the session alive would be useful?

Reply:
No, I don't have any use case in mind for keeping the session alive. I wanted to keep the session by default and stop the session when the configuration is used. However, it seems like the new configuration key is not necessary as stopping session can be the default behavior👍. I personally think, evcc should be terminated as a default after session, because evcc is not starting a new session again.